packages/node/src/node.ts

```diff
@@ -118,9 +119,13 @@ export class Node {
           );
         }
 
+        const signingKey = new SigningKey(
+          this.signer.derivePath("0").privateKey
```

Comment:
Shouldn't this be the extended signing key for the AppInstance that this commitment is about?

Reply:
So, we actually sign every commitment using the 0th key (_except_ for `UpdateProtocol` commitments, in which case you're right this was a mistake on my part). However, the Update Protocol could also just make a call to the multisig instead of to the `AppRegistry` then we wouldn't need to change this code.

Reply:
Because every commitment except `SetStateCommitment` has `to` being `multisigAddress` and so we verify the signature based on `multisigOwners` which is the 0th key.

Reply:
So the public signing keys used per AppInstance are unique, but are derived from a higher xkey that can be used to sign commitments, resulting in a signature whose recovered address is the Node's xpub address.

Right?